File: golang-for-nodejs-developers/examples/arrays.js
const array = [1, 2, 3, 4, 5]
console.log(array)

const clone = [...array]
console.log(clone)

const sub = array.slice(2, 4)
console.log(sub)

const concatenated = [...clone, 6, 7]
console.log(concatenated)

const prepended = [-2, -1, 0, ...concatenated]
console.log(prepended)

File: golang-for-nodejs-developers/examples/arrays.go
package main

import (
	"fmt"
	"slices"
)

func main() {
	array := []int{1, 2, 3, 4, 5}
	fmt.Println(array)

	clone := slices.Clone(array)
	fmt.Println(clone)

	sub := array[2:4]
	fmt.Println(sub)

	concatenated := slices.Concat(array, []int{6, 7})
	fmt.Println(concatenated)

	prepended := slices.Concat([]int{-2, -1, 0}, concatenated)
	fmt.Println(prepended)
}
